File: uber-front/src/app/private/models/ReportDTO.ts
export class ReportDTO {

  public startDate: Date;
  public endDate: Date;
  public userId: number | null;

  constructor(startDate: Date, endDate: Date, userId: number | null) {
    this.startDate = startDate;
    this.endDate = endDate;
    this.userId = userId;
  }



}

File: uber/src/main/java/com/nwtkts/uber/dto/ReportDTO.java
package com.nwtkts.uber.dto;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
public class ReportDTO {

    //@JsonFormat(pattern="yyyy-MM-dd'T'HH:mm:ss")
    private LocalDateTime startDate;

    //@JsonFormat(pattern="yyyy-MM-dd'T'HH:mm:ss")
    private LocalDateTime endDate;

    private Long userId;

}
